installers/pgo-installer/ansible/roles/pgo-installer/tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+---
+- name: Set output directory fact
+  set_fact:
+    output_dir: "{{ ansible_env.HOME }}/.pgo/{{ pgo_operator_namespace }}/output"
+  tags: 
+    - always
+
+- name: Ensure output directory exists
+  file:
+    path: "{{ output_dir }}"
+    state: directory
+    mode: 0700
+  tags:
+    - always
+
+- include_vars: openshift.yml
+  when: use_oc | default(false) | bool
+  tags:
+    - always
```

Comment:
I think we need a different flag or the like, given we'll use `kubectl` instead of oc.

The suggestion I had in an earlier comment:

`pgo_installer_environment="openshift"` so we check here if `pgo_installer_environment` containers the value `openshift`

Reply:
This will be what they are using locally so we won't have control over it. Beyond telling users which to use. I'm not sure `pgo_installer_environemtn` would do a good enough job at letting the user know this is what is used locally.